### ISSUE TITLE
Handling of legacy CHANGELOG.md files

### ISSUE THREAD

**Issue by DennisDobslaf** [churchtools/changelogger#28]
I used this tool on a repo with an existing but old CHANGELOG.md file with no common schema. 
This tool doesn't write to may legacy CHANGELOG.md on changelogger release <tag> but deletes the unreleased files.

**Comment by DennisDobslaf** [comment]
Self-answering my question:

here is the reason: https://github.com/churchtools/changelogger/blob/master/app/Commands/ReleaseCommand.php#L103

As a workaround one can insert `<!-- CHANGELOGGER -->` at the top of a legacy CHANGELOG.md file.

**Comment by obstschale** [comment]
Yes, the tool looks for `<!-- CHANGELOGGER -->` and uses this as placeholder. I will document this. Thx for pointing this out.

**Comment by strausmann** [comment]
Thanks for the tip, i had the same problem.
